internal/buildscripts/packaging/msi/splunk-otel-collector.wxs

```diff
@@ -67,6 +67,7 @@
 
       <!-- Copy the default agent config file to ProgramData if it does not already exist -->
       <CustomAction Id="CopyConfig" ExeCommand="xcopy /y &quot;[INSTALLDIR]agent_config.yaml&quot; &quot;[CommonAppDataFolder]Splunk\OpenTelemetry Collector\agent_config.yaml*&quot;" Directory="INSTALLDIR" Impersonate="no" Execute="deferred" Return="check" />
+      <!-- Copy the JMX Metric Gatherer JAR to opt if it does not already exist -->
```

Comment:
```suggestion
```
Also, you may need to include `opt` in `<Directory>` above to package and install the jar since it's not getting copied anymore from `Program Files`.

Reply:
I'm not following. Can you explain what you mean by including `opt` in `<Directory>` above? I understand it's not getting copied properly, but I'm not seeing how to fix that any way more simply than what was being done originally:
```
 <CustomAction Id="CopyJMXJAR" ExeCommand="xcopy /y &quot;[INSTALLDIR]opentelemetry-java-contrib-jmx-metrics.jar&quot; &quot;[WindowsVolume]\opt\opentelemetry-java-contrib-jmx-metrics.jar*&quot;" Directory="INSTALLDIR" Impersonate="no" Execute="deferred" Return="check" />
```

Should I take that solution and modify it for the current paths involved?

Reply:
When we package/install the other files in `\Program Files`, we define the source and target paths in https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/splunk-otel-collector.wxs#L14.  Since the jar is not currently included, I think we'll have to set the source path in https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/make.ps1#L62 add a new `Directory` element in the wxs file.

Reply:
> When we package/install the other files in `\Program Files`, we define the source and target paths in https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/splunk-otel-collector.wxs#L14. Since the jar is not currently included, I think we'll have to set the source path in ~~https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/make.ps1#L62~~ add a new `Directory` element in the wxs file.

Sorry, I referenced the file, it should be https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/msi-builder/build.sh#L166